tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js

```diff
@@ -59,20 +59,57 @@ describe('PopupTabLayout', function () {
 				});
 
 				it('should expand the tabs when focus returns to the tabs', function () {
-					Page.waitTransitionEnd(1500, 'waiting for Panel transition', () => {
-						Page.spotlightRight();
-						Page.spotlightSelect();
-					});
-					Page.waitTransitionEnd(1500, 'waiting for Panel transition', () => {
-						Page.spotlightLeft();
-					});
+					Page.spotlightRight();
```

Comment:
it need to be verified on Jenkins

Reply:
This line does not introduce any transition now since it's changed not to.
I've added `delay` when the transition is needed below.
I ran all the tests on jenkins and the tests are all passed from #1565.